src/dashboard/apps/core/models.py

```diff
@@ -194,3 +194,26 @@ class Meta:  # noqa: D106
 
     def __str__(self):  # noqa: D105
         return self.provider_assigned_id
+
+
+class Station(DashboardBase):
+    """Represents a station for electric vehicles."""
+
+    id_station_itinerance = models.CharField(
+        _("id station itinerance"), max_length=35, blank=True, unique=True
```

Comment:
The `id_station_itinerance` should be `unique_together` with the `station_name`.

This is done in PR #496 ([here](https://github.com/MTES-MCT/qualicharge/blob/d0ffb7d7cdf45bef83fc615026469b20f44c270b/src/dashboard/apps/core/models.py#L267))

Reply:
That's odd to have two stations with the same `id_station_itinerance`. We should discuss this tomorrow.